src/app/pages/FeatureIdxPage/index.test.jsx

```diff
@@ -110,7 +110,7 @@ jest.mock('#containers/PageHandlers/withContexts', () => Component => {
   );
 });
 
-describe('Feature Idx Page', () => {
+describe.skip('Feature Idx Page', () => {
```

Comment:
If we get the go ahead to go live with this, we will then delete the FeatureIdxPage code & supporting utils/tests in a separate PR